receiver/awscontainerinsightreceiver/internal/neuron/neuron_empty_metric_decorator.go

```diff
@@ -0,0 +1,159 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package neuron
+
+import (
+	"context"
+	"strconv"
+	"strings"
+
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+	"go.uber.org/zap"
+)
+
+const (
+	statusType     = "status_type"
+	errorType      = "error_type"
+	memoryLocation = "memory_location"
+	percentile     = "percentile"
+)
+
+var attributeConfig = map[string][]string{
+	NeuronExecutionStatus:                       {statusType},
+	NeuronExecutionErrors:                       {errorType},
+	NeuronRuntimeMemoryUsage:                    {memoryLocation},
+	NeuronExecutionLatency:                      {percentile},
+	NeuronCoreUtilization:                       {neuronCoreAttributeKey, neuronDeviceAttributeKey},
+	NeuronCoreMemoryUtilizationConstants:        {neuronCoreAttributeKey, neuronDeviceAttributeKey},
+	NeuronCoreMemoryUtilizationModelCode:        {neuronCoreAttributeKey, neuronDeviceAttributeKey},
+	NeuronCoreMemoryUtilizationSharedScratchpad: {neuronCoreAttributeKey, neuronDeviceAttributeKey},
+	NeuronCoreMemoryUtilizationRuntimeMemory:    {neuronCoreAttributeKey, neuronDeviceAttributeKey},
+	NeuronCoreMemoryUtilizationTensors:          {neuronCoreAttributeKey, neuronDeviceAttributeKey},
+}
+
+var nonCoreAttributeValues = map[string]string{
+	statusType:     "completed",
+	errorType:      "generic",
+	memoryLocation: "neuron_device",
+	percentile:     "p50",
+}
+
+type EmptyMetricDecorator struct {
```

Comment:
A comment explaining why we need to have this decorator and what it does would be useful.